### ISSUE TITLE
If I log with one account, everything works, but if I use a different account, I get 'NoneType object is not subscriptable'?

### ISSUE THREAD

**Issue by Javieralonb** [instaloader/instaloader#1117]
I have an account that I have been using for a while and I never had any problem. Now I need to extract more data and I'm getting the 'too many requests' error. So I figured out that I could add a different account to my daily process in order to share the load between 2 accounts. I don't even know if this is viable. 
The thing is this new account I created gets 'NoneType object is not subscriptable' when I try to download anything. I'm executing the exact same code, but changing the account changes the results.  What could be the reason?
Thank you for your answers.

**Comment by Showmaster3000** [comment]
you didn't write about your command line or error message, so helping is nearly impossible. But it looks like duplicate of #1109

**Comment by Javieralonb** [comment]
This code, with the user 'aaaaa', works perfectly.

```
USER = 'aaaaa'
PASSWORD= 'aaaaa'
L = instaloader.Instaloader()
L.login(USER, PASSWORD)   
profile = Profile.from_username(L.context, 'x')
for post in profile.get_tagged_posts():
      shortcode = post.shortcode
```
This code, with the user 'bbbbb' returns 'NoneType object is not subscriptable' error.

```
USER = 'bbbbb'
PASSWORD= 'bbbbb'
L = instaloader.Instaloader()
L.login(USER, PASSWORD)   
profile = Profile.from_username(L.context, 'x')
for post in profile.get_tagged_posts():
      shortcode = post.shortcode
```

The only thing that has changed is the account used to log. Different accounts give different results for some reason, when the operation is identical. This only happened to me with get_tagged_posts(). The acount 'bbbbb' seem to work fine for other operations like get_posts(). The target barely has 2 or 3 tagged posts, so it should be an easy and fast operation.
The account 'aaaaa' is like 5 months old. It has a phone number added, since I had to in order to keep pulling data. The account 'bbbbb' does not have a number added yet. This would be the only difference. 
I hope I was able to explain the issue properly, English is not my native language.
